### ISSUE TITLE
Convert examples into ProtoSchool Tutorials

### ISSUE THREAD

**Issue by daviddias** [ipfs/js-ipfs#1816]
Perhaps they won't all be doable on ProtoSchool, but going through the list I would say that at least we can:

- https://github.com/ipfs/js-ipfs/tree/master/examples/browser-mfs
- https://github.com/ipfs/js-ipfs/tree/master/examples/browser-add-readable-stream
- https://github.com/ipfs/js-ipfs/tree/master/examples/browser-readablestream
- https://github.com/ipfs/js-ipfs/tree/master/examples/browser-video-streaming
- https://github.com/ipfs/js-ipfs/tree/master/examples/custom-libp2p
- https://github.com/ipfs/js-ipfs/tree/master/examples/explore-ethereum-blockchain
- https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-101
- https://github.com/ipfs/js-ipfs/tree/master/examples/traverse-ipld-graphs

**Comment by daviddias** [comment]
These could actually be tutorials to get help from volunteers that are looking for ideas of tutorials to write

**Comment by alanshaw** [comment]
I agree we should re-use some of the work in these examples to create some awesome protoschool tutorias. Shall we move this issue to https://github.com/ProtoSchool/protoschool.github.io/issues?

**Comment by alanshaw** [comment]
@terichadbourne I can't actually transfer this issue. Is it useful if I open an issue on protoschool with David's comment above?

**Comment by terichadbourne** [comment]
@alanshaw It would be great to move this discussion to the ProtoSchool repo with a `new-tutorial` tag, but I'm a bit concerned it will have fewer eyes on it there for brainstorming. Shall we leave the issue open here with a link requesting the convo be continued there? Get at least a couple of folks from the js-ipfs team tagged over there so they can stay involved in advising? What do you recommend?

Ultimately we'll want separate issues in that repo for each potential tutorial, but I'm okay with starting with a single issue like this one as we sort out what's most viable in the ProtoSchool format.

**Comment by alanshaw** [comment]
Please continue conversation here: https://github.com/ProtoSchool/protoschool.github.io/issues/136

**Comment by achingbrain** [comment]
Closing as this is a protoschool issue
